cmd/zc_attr_filter_windows.go

```diff
@@ -0,0 +1,165 @@
+// Copyright © 2017 Microsoft <wastore@microsoft.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"path/filepath"
+	"syscall"
+	"strings"
+)
+
+type excludeAttrFilter struct {
+	fileAttributes uint32
+	filePath       string
+}
+
+func (f *excludeAttrFilter) doesSupportThisOS() (msg string, supported bool) {
+	msg = ""
+	supported = true
+	return
+}
+
+func (f *excludeAttrFilter) doesPass(storedObject storedObject) bool {
+	fileName := filepath.Join(f.filePath, storedObject.name)
```

Comment:
Can you add a comment above this line, saying the following (to remind us later when we are working on long-path support)
```
TODO: review handling of filename for long path support
```

There's no point changing any code now, because we have not finalized exactly how we'll do the long path support (it's partially there already in the Go SDK, but there are unsupported edge cases)

Reply:
Done.